src/target/llvm/codegen_llvm.cc

```diff
@@ -685,11 +685,11 @@ llvm::Value* CodeGenLLVM::CreateCallExtern(const CallNode* op) {
   }
   llvm::FunctionType* ftype = llvm::FunctionType::get(
       GetLLVMType(GetRef<PrimExpr>(op)), arg_type, false);
-  llvm::Function* f = module_->getFunction(op->name);
+  llvm::Function* f = module_->getFunction((std::string)op->name);
```

Comment:
we should be able to use `op->name.c_str()` or `op->name.operator std::string()` instead of using this C-style explicit casting

Reply:
Hi @zhiics ,

Thanks for your kind reminder. I change the C-style explicit casting with `op->name.c_str()`.